### ISSUE TITLE
Warning: autotrain.trainers.common:__init__:176 - Parameters supplied but not used: data_path

### ISSUE THREAD

**Issue by 11shavkat** [huggingface/autotrain-advanced#584]
I was trying to use autotrain on google colab and i got this error when i start training:

> Device 0: Tesla T4 - 260.9MiB/15360MiB

-----------------

INFO | 2024-04-13 15:23:39 | autotrain.app_utils:get_running_jobs:33 - Process 2603 is already completed. Skipping...

INFO | 2024-04-13 15:23:39 | autotrain.app_utils:get_running_jobs:32 - Error while killing process: [Errno 3] No such process

INFO | 2024-04-13 15:23:39 | autotrain.app_utils:get_running_jobs:28 - Killing PID: 2603

{'latents_mean', 'latents_std'} was not found in config. Values will be initialized to default values.

{'variance_type', 'thresholding', 'dynamic_thresholding_ratio', 'clip_sample_range'} was not found in config. Values will be initialized to default values.

You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.

You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.

INFO | 2024-04-13 15:21:10 | autotrain.backend:create:305 - Training PID: 2603

INFO | 2024-04-13 15:21:10 | autotrain.commands:launch_command:339 - {'model': 'stablediffusionapi/epicrealism-xl', 'vae_model': '', 'revision': None, 'tokenizer': None, 'image_path': 'autotrain-tp7lr-fasg3/autotrain-data', 'class_image_path': None, 'prompt': '<gina>', 'class_prompt': None, 'num_class_images': 100, 'class_labels_conditioning': None, 'prior_preservation': False, 'prior_loss_weight': 1.0, 'project_name': 'autotrain-tp7lr-fasg3', 'seed': 42, 'resolution': 1024, 'center_crop': False, 'train_text_encoder': False, 'batch_size': 1, 'sample_batch_size': 4, 'epochs': 1, 'num_steps': 500, 'checkpointing_steps': 500, 'resume_from_checkpoint': None, 'gradient_accumulation': 4, 'disable_gradient_checkpointing': False, 'lr': 0.0001, 'scale_lr': False, 'scheduler': 'constant', 'warmup_steps': 0, 'num_cycles': 1, 'lr_power': 1.0, 'dataloader_num_workers': 0, 'use_8bit_adam': False, 'adam_beta1': 0.9, 'adam_beta2': 0.999, 'adam_weight_decay': 0.01, 'adam_epsilon': 1e-08, 'max_grad_norm': 1.0, 'allow_tf32': False, 'prior_generation_precision': None, 'local_rank': -1, 'xformers': False, 'pre_compute_text_embeddings': False, 'tokenizer_max_length': None, 'text_encoder_use_attention_mask': False, 'rank': 4, 'xl': False, 'mixed_precision': 'fp16', 'token': '*****', 'repo_id': 'bobitalent/autotrain-tp7lr-fasg3', 'push_to_hub': True, 'username': 'bobitalent', 'validation_prompt': None, 'num_validation_images': 4, 'validation_epochs': 50, 'checkpoints_total_limit': None, 'validation_images': None, 'logging': False}

INFO | 2024-04-13 15:21:10 | autotrain.commands:launch_command:338 - ['python', '-m', 'autotrain.trainers.dreambooth', '--training_config', 'autotrain-tp7lr-fasg3/training_params.json']

INFO | 2024-04-13 15:21:10 | autotrain.backend:create:300 - Starting local training...

WARNING | 2024-04-13 15:21:10 | autotrain.trainers.common:__init__:176 - Parameters supplied but not used: data_path

WARNING | 2024-04-13 15:21:10 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: local_rank, scale_lr, center_crop, xl, adam_epsilon, train_text_encoder, validation_prompt, num_validation_images, lr_power, prior_generation_precision, class_labels_conditioning, sample_batch_size, logging, text_encoder_use_attention_mask, adam_weight_decay, allow_tf32, adam_beta1, validation_images, revision, epochs, validation_epochs, dataloader_num_workers, class_image_path, prior_preservation, pre_compute_text_embeddings, warmup_steps, resume_from_checkpoint, seed, disable_gradient_checkpointing, tokenizer, num_class_images, prior_loss_weight, adam_beta2, rank, tokenizer_max_length, checkpoints_total_limit, max_grad_norm, class_prompt, num_cycles

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abd960>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abf550>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abfd00>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abe440>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab58eea03d0>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab58eea1120>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab58eea1060>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abe110>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abe140>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abe770>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abe410>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abe2f0>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abf370>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abee60>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abf160>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abeb60>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:63 - <starlette.datastructures.UploadFile object at 0x7ab595abdc00>

INFO | 2024-04-13 15:21:10 | autotrain.preprocessor.dreambooth:_save_concept_images:62 - Saving concept images

INFO | 2024-04-13 15:21:09 | autotrain.app:handle_form:435 - hardware: Local

INFO | 2024-04-13 15:20:39 | autotrain.app:fetch_params:206 - Task: dreambooth

INFO | 2024-04-13 15:20:29 | autotrain.app:fetch_params:206 - Task: llm:sft

INFO | 2024-04-13 15:20:23 | autotrain.app:<module>:152 - AutoTrain started successfully

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: token, save_total_limit, model, batch_size, valid_split, save_strategy, weight_decay, epochs, gradient_accumulation, username, repo_id, max_seq_length, seed, warmup_ratio, lr, tokens_column, scheduler, logging_steps, train_split, tags_column, max_grad_norm, auto_find_batch_size, evaluation_strategy, optimizer, project_name, data_path, push_to_hub

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: local_rank, checkpointing_steps, scale_lr, center_crop, model, xl, adam_epsilon, repo_id, validation_prompt, num_validation_images, lr_power, prior_generation_precision, num_cycles, class_labels_conditioning, sample_batch_size, logging, text_encoder_use_attention_mask, adam_weight_decay, adam_beta1, allow_tf32, token, validation_images, revision, epochs, dataloader_num_workers, validation_epochs, username, prior_preservation, pre_compute_text_embeddings, warmup_steps, resume_from_checkpoint, seed, image_path, tokenizer, num_class_images, prior_loss_weight, scheduler, adam_beta2, tokenizer_max_length, rank, checkpoints_total_limit, max_grad_norm, class_prompt, class_image_path, project_name, push_to_hub

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: token, id_column, model, valid_split, categorical_columns, username, repo_id, target_columns, seed, numerical_columns, num_trials, train_split, task, time_limit, project_name, data_path, push_to_hub

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: lora_r, model, weight_decay, gradient_accumulation, repo_id, max_seq_length, target_column, warmup_ratio, lr, max_target_length, logging_steps, text_column, token, save_total_limit, valid_split, batch_size, epochs, username, seed, scheduler, lora_dropout, quantization, train_split, lora_alpha, max_grad_norm, auto_find_batch_size, evaluation_strategy, optimizer, project_name, peft, data_path, push_to_hub

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: token, save_total_limit, model, batch_size, valid_split, save_strategy, weight_decay, epochs, gradient_accumulation, username, repo_id, target_column, seed, warmup_ratio, image_column, lr, scheduler, logging_steps, train_split, max_grad_norm, auto_find_batch_size, evaluation_strategy, optimizer, project_name, data_path, push_to_hub

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: token, save_total_limit, model, batch_size, valid_split, save_strategy, weight_decay, epochs, gradient_accumulation, username, repo_id, max_seq_length, target_column, seed, warmup_ratio, lr, scheduler, logging_steps, train_split, text_column, max_grad_norm, auto_find_batch_size, evaluation_strategy, optimizer, project_name, data_path, push_to_hub

WARNING | 2024-04-13 15:20:14 | autotrain.trainers.common:__init__:170 - Parameters not supplied by user and set to default: dpo_beta, lora_r, model, weight_decay, gradient_accumulation, repo_id, model_max_length, prompt_text_column, rejected_text_column, warmup_ratio, merge_adapter, use_flash_attention_2, lr, logging_steps, text_column, model_ref, token, save_total_limit, valid_split, batch_size, username, seed, disable_gradient_checkpointing, trainer, scheduler, lora_dropout, train_split, lora_alpha, add_eos_token, max_grad_norm, evaluation_strategy, auto_find_batch_size, optimizer, project_name, data_path, push_to_hub

INFO | 2024-04-13 15:20:14 | autotrain.app:<module>:30 - Starting AutoTrain...

Your installed package `nvidia-ml-py` is corrupted. Skip patch functions `nvmlDeviceGetMemoryInfo`. You may get incorrect or incomplete results. Please consider reinstall package `nvidia-ml-py` via `pip3 install --force-reinstall nvidia-ml-py nvitop`.

Your installed package `nvidia-ml-py` is corrupted. Skip patch functions `nvmlDeviceGet{Compute,Graphics,MPSCompute}RunningProcesses`. You may get incorrect or incomplete results. Please consider reinstall package `nvidia-ml-py` via `pip3 install --force-reinstall nvidia-ml-py nvitop`.

**Comment by abhishekkrthakur** [comment]
is it happening with normal base models? sdxl1, 2, etc?

**Comment by 11shavkat** [comment]
> is it happening with normal base models? sdxl1, 2, etc?

im using epcirealism

**Comment by abhishekkrthakur** [comment]
have you tried sdxl? was epicrealism working before?

**Comment by 11shavkat** [comment]
> have you tried sdxl? was epicrealism working before?

no i didnt, i was thinking to use epicrealism because i want to create an ai influencer and i want to look as real as possible. i tried on rendernet epicrealism and i saw the results are pretty good and seems real.
do u have a better idea tho?
